docs/pages/access-controls/sso/google-workspace.mdx

```diff
@@ -4,228 +4,302 @@ description: How to configure Teleport access with Google Workspace (formerly kn
 videoBanner: WTLWc6nnPfk
 ---
 
-This guide will explain how to configure [Google Workspace](https://workspace.google.com/)
-to be a single sign-on (SSO) provider to issue Teleport credentials to specific
-groups of users. When used in combination with role-based access control (RBAC)
-it allows Teleport administrators to define policies like:
+This guide explains how to configure [Google Workspace](https://workspace.google.com/)
+to be a single sign-on (SSO) provider that issues Teleport credentials to specific
+groups of users. When you use Google Workspace in combination with Teleport role-based 
+access control (RBAC), you can define policies like the following:
 
 - Only members of the "DBA" Google group can connect to PostgreSQL databases.
-- Developers must never SSH into production servers.
+- Developers must never use SSH to access production servers.
```

Comment:
I think using SSH as a verb is commonly [accepted](https://en.wiktionary.org/wiki/ssh).